app/controllers/api/v2/providers_controller.rb

```diff
@@ -125,27 +123,22 @@ def get_user
       def update_accrediting_enrichment
         return if accredited_bodies_params.values.none?
 
-        enrichment = @provider.enrichments.find_or_initialize_draft(current_user)
-
-        enrichment.accrediting_provider_enrichments =
+        @provider.accrediting_provider_enrichments =
           accredited_bodies_params["accredited_bodies"].map do |accredited_body|
             {
               UcasProviderCode: accredited_body["provider_code"],
               Description: accredited_body["description"],
             }
           end
 
-        enrichment.save
+        @provider.save
       end
 
       def update_enrichment
```

Comment:
This is going to be confusing to our future selves. Can we rename it so something that doesn't even mention `enrichment`?  For example in `CourseController` we use `update_course`.

Reply:
sorted